### DISCUSSION TITLE
Does FBX support

### DISCUSSION THREAD

**Discussion by eherozhao** [assimp/assimp#3785] [Q&A] [answered]
In the [introduction](https://assimp-docs.readthedocs.io/en/latest/about/introduction.html), there is no FBX format support, but I found fbx exporter in the source code. Does assimp still officially support FBX import and export now? Thank you.

**Comment by kimkulling** [comment, marked as answer]
Good catch, I just saw we have forgotten to add this format. This issue is fixed right now.

**Reply by eherozhao** [reply]
Hello, I find a typo in the document. The gltf is the right data type I suppose, but you use glft. Please check it again. I planed to create a PR about it but it was minor.
Also, will you support USD in the future? Thank you.
